README.md

```diff
@@ -8,7 +8,7 @@ For Commitments V2, see: https://github.com/SkillsFundingAgency/das-commitments/
 ## Getting started Api (covering v1 and v2) ##
 * Clone das-commitments repo
 * Open das-commitments solution - build fails. This is due to a Slow Cheetah issue. To workaround, change build configuration to Release and build, then back to Debug and build. Solution will then build ok.
-* Set Startup projects as Multiple - SFA.DAS.CloudService, SFA.DAS.CommitmentsV2.Api (this is for Commitments v1 and v2 respectively)
+* Run the SFA.DAS.CommitmentsV2.Api project (this is for Commitments v1 and v2 respectively)
```

Comment:
Remove the text in brackets as no longer relevant: `(this is for Commitments v1 and v2 respectively)`

Reply:
Done